config.yml

```diff
@@ -8,7 +8,7 @@ defaults:
     rabbitQueueExpirationDelay: '- 24 hours' # This means we re-send alerts at most once a day
 
     # namespaces are rotated at this time interval.
-    namespaceRotationInterval: '1 hour'
+    namespaceRotationInterval: '6 hours'
```

Comment:
Why?

Just because the username/password has been rotated doesn't mean that we have to disconnect in `pulse-publisher`.

taskcluster-pulse should give `pulse-publisher` an object with rabbitmq connections string and expiration time for when it's no longer valid.... `pulse-publisher` should connect and use said connection until it's broken (for some reason), when the connection breaks `pulse-publisher` reconnects using previous connection string, unless it is past the expiration time in which case it calls taskcluster-pulse.

The fact that taskcluster-pulse rotates passwords is not a reason for `pulse-publisher` to disconnect...